packages/link/src/components/Link.story.js

```diff
@@ -2,11 +2,10 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 
+import { fontSizes } from "@crave/farmblocks-theme";
```

Comment:
We usually would group and move this kind of import to line 4. Any reason to place it here?

Reply:
This line was auto fixed by eslint